client-scripts-api.md

```diff
@@ -277,3 +277,111 @@ Payload:
   value: true|false
 }
 ```
+
+### `routeChange`
+
+This event is triggered when a change in the internal router happens.
+
+Payload:
+
+```js
+{
+  location: {
+    pathname: '/products',
+    search: '?page=0',
+    query: {
+      page: 0,
+    },
+    hash: '#',
+  },
+  action: 'PUSH',
```

Comment:
❓ can you explain what action means and what other values are to be expected?

Reply:
it's the history action from the browser API. so PUSH means it pushes to the history. REPLACE means it replaces the last item from the history (like a redirect or just a small change like adding a search param that should not result in a new history entry.

Reply:
still maybe good to add in comment which values are possible in enums like this ```// PUSH, REPLACE```

Reply:
Yeah, I will add it